soccer/gameplay/plays/offense/adaptive_formation.py

```diff
@@ -145,6 +145,8 @@ def __init__(self):
 
     @classmethod
     def score(cls):
```

Comment:
Its the score that you see to the right of the play in the simulator. If you have multiple plays running at once, I think it selects the highest score or something.